### PR TITLE
added function return_outputs() which replaces demisto.results()

### PR DESCRIPTION
**Pull request by anara123**
## Status
Ready

## Description
return_outputs() which replaces demisto.results() - more return outputs in demisto should be more intuitive 

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests - no tests - because it does demisto.results() and not returns anything
- [x] Documentation - will documented in Script Helper automatically
- [ ] Code Review

### CODE REVIEW

File: Scripts/script-CommonServerPython.yml [hunk 1]
```diff
@@ -1203,6 +1203,46 @@ script: |-
       return True
 
 
+  def return_outputs(readable_output, outputs, raw_response=None):
+      """
+      This function replaces demisto.results(), makes the use more easy and intuitive.
```
Comment (ronykoz) [inline Scripts/script-CommonServerPython.yml]: This function wraps the demisto.results(), makes the usage of returning results to the user more intuitively.

File: Scripts/script-CommonServerPython.yml [hunk 2]
```diff
@@ -1203,6 +1203,46 @@ script: |-
...
+      :return: None
+      :rtype: ``None``
+      """
+      return_entry = {
+          "Type": entryTypes["note"],
+          "HumanReadable": "",
+          "ContentsFormat": formats["text"],
+          "Contents": None
+      }
+      if readable_output:
```
Comment (ronykoz) [inline Scripts/script-CommonServerPython.yml]: No need, it's mandatory as is, just put it directly into the dict

File: Scripts/script-CommonServerPython.yml [hunk 3]
```diff
@@ -1203,6 +1203,46 @@ script: |-
...
+          return_entry["HumanReadable"] = readable_output
+
+      if raw_response is not None:
+          return_entry["ContentsFormat"] = formats["json"]
+          return_entry["Contents"] = raw_response
+
+      if outputs is not None:
```
Comment (ronykoz) [inline Scripts/script-CommonServerPython.yml]: same here

File: Scripts/script-CommonServerPython.yml [hunk 4]
```diff
@@ -1203,6 +1203,46 @@ script: |-
...
+      :param raw_response: must be dictionary, if not provided then will be equal to outputs. usually must be the original
+      raw response from the 3rd party service (originally Contents)
+
+      :return: None
+      :rtype: ``None``
+      """
+      return_entry = {
+          "Type": entryTypes["note"],
+          "HumanReadable": "",
+          "ContentsFormat": formats["text"],
```
Comment (ronykoz) [inline Scripts/script-CommonServerPython.yml]: i think it should always be json, you are changing it to json anyway..

**Comment by anara123** [comment]
@ronykoz fixed

**Pull request merged**